### PR TITLE
Fix some bug in lbp page

### PR DESCRIPTION
**Pull request by leonthuongto**
## Description

Please describe the purpose of this pull request.

## Changes

- Refactor show time remaining

## Attached Links

1. Jira ticket:
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/utils/time.ts [hunk 1]
```diff
@@ -144,3 +144,24 @@ export const isAfterWithSameTime = (from: Moment, to: Moment | null) => {
...
+        return '0 Hour'
+      }
+      return 'Less than 1 Hour'
+    }
+    if (remainingHours === 1) {
+      return '1 Hour'
+    }
+    return `${remainingHours} Hours`
+  }
+}
```
Comment (thi-investax) [inline src/utils/time.ts:167]: Can we refactor this if else statement? could leverage chat GPT

File: src/utils/time.ts [hunk 2]
```diff
@@ -145,23 +145,20 @@ export const toUnixTimeSeconds = (date: Date): number => {
...
 }
 
-export const displayRemainingTime = (remainingDays: number, remainingHours: number) => {
+export const displayRemainingTime = (remainingDays: number, remainingHours: number): string => {
+  // Handle days first
   if (remainingDays > 1) {
-    return `${remainingDays} Days`
+    return `${remainingDays} Days`;
+  } else if (remainingDays === 1) {
+    return '1 Day';
```
Comment (thi-investax) [inline src/utils/time.ts:153]: hmm looks good now but are you using prettier @leonthuongto ? saw the semi colons

Comment (leonthuongto) [reply src/utils/time.ts]: Done bro

Comment (thi-investax) [reply src/utils/time.ts]: LGTM!

**Pull request merged**